cmd/influxd/launcher/cmd.go

```diff
@@ -205,11 +205,11 @@ func NewOpts(viper *viper.Viper) *InfluxdOpts {
 
 		NoTasks: false,
 
-		ConcurrencyQuota:                0,
+		ConcurrencyQuota:                1024,
```

Comment:
Were you going to lower this because of the goproc limit?

Reply:
I think you saw in the original PR, but: the limits are lowered in the test launcher. See `launcher_helpers.go`